CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.33.0
+### Notable Changes
+* Migrate CreateVolume response topology to standard label topology.kubernetes.io/zone ([#2086](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2086), [@ConnorJC3](https://github.com/ConnorJC3))
+* Modifying Tags On Existing Volume ([#2082](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2082), [@mdzraf](https://github.com/mdzraf))
+* Add --kubeconfig flag for out-of-cluster auth ([#2081](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2081), [@cartermckinnon](https://github.com/cartermckinnon))
+
+### Bug Fixes
+* Bump GCR sidecars that reference broken tags ([#2091](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2091), [@ConnorJC3](https://github.com/ConnorJC3))
+* govulncheck: Bump google.golang.org/grpc to v1.64.1 ([#2085](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2085), [@ConnorJC3](https://github.com/ConnorJC3))
+* Change coalescer InputType from comparable to any ([#2083](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2083), [@ConnorJC3](https://github.com/ConnorJC3))
+* Bump go version to fix govulncheck failure ([#2080](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2080), [@ConnorJC3](https://github.com/ConnorJC3))
+* Use new client token when CreateVolume returns IdempotentParameterMismatch ([#2075](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2075), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
+* fix function name in comment #2088 ([#2088](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2088), [@augustkang](https://github.com/augustkang))
```

Comment:
Please capitalize this F. It's okay for Changelog sentence to be different from PR title. (Usually we pick a more customer friendly summary for changelog)